guide/english/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md

```diff
@@ -3,8 +3,7 @@ title: Moving Forward From Here
 ---
 ## Moving Forward From Here
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/front-end-libraries/react-and-redux/moving-forward-from-here/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
-
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+### Solution
```

Comment:
Please change this one to the details and summary thing you've done for all challenges

Reply:
I just pushed this change up.